### PR TITLE
Add wrapping for 100 columns text width.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
After build validation for right margin trespassing, this is PR that contain fixes in wrapping.

### CODE REVIEW

File: client/src/main/java/org/spine3/net/Patterns.java [hunk 1]
```diff
@@ -60,103 +60,120 @@
...
         + "|\u0639\u0631\u0627\u0642|\u0639\u0645\u0627\u0646|\u0641\u0644\u0633\u0637\u064a\u0646"
-        + "|\u0642\u0637\u0631|\u0643\u0648\u0645|\u0645\u0635\u0631|\u0645\u0644\u064a\u0633\u064a\u0627"
+        + "|\u0642\u0637\u0631|\u0643\u0648\u0645|\u0645\u0635\u0631"
+        + "|\u0645\u0644\u064a\u0633\u064a\u0627"
         + "|\u0645\u0648\u0642\u0639|\u0915\u0949\u092e|\u0928\u0947\u091f|\u092d\u093e\u0930\u0924"
-        + "|\u0938\u0902\u0917\u0920\u0928|\u09ad\u09be\u09b0\u09a4|\u0a2d\u0a3e\u0a30\u0a24|\u0aad\u0abe\u0ab0\u0aa4"
-        + "|\u0b87\u0ba8\u0bcd\u0ba4\u0bbf\u0baf\u0bbe|\u0b87\u0bb2\u0b99\u0bcd\u0b95\u0bc8|\u0b9a\u0bbf\u0b99\u0bcd\u0b95\u0baa\u0bcd\u0baa\u0bc2\u0bb0\u0bcd"
-        + "|\u0c2d\u0c3e\u0c30\u0c24\u0c4d|\u0dbd\u0d82\u0d9a\u0dcf|\u0e04\u0e2d\u0e21|\u0e44\u0e17\u0e22"
+        + "|\u0938\u0902\u0917\u0920\u0928|\u09ad\u09be\u09b0\u09a4|\u0a2d\u0a3e\u0a30\u0a24"
+        + "|\u0aad\u0abe\u0ab0\u0aa4"
```
Comment (dmdashenkov) [inline client/src/main/java/org/spine3/net/Patterns.java]: This line seems strangely short.

File: client/src/main/java/org/spine3/net/Patterns.java [hunk 2]
```diff
@@ -60,103 +60,120 @@
...
+        + "|\u0b87\u0ba8\u0bcd\u0ba4\u0bbf\u0baf\u0bbe|\u0b87\u0bb2\u0b99\u0bcd\u0b95\u0bc8"
+        + "|\u0b9a\u0bbf\u0b99\u0bcd\u0b95\u0baa\u0bcd\u0baa\u0bc2\u0bb0\u0bcd"
+        + "|\u0c2d\u0c3e\u0c30\u0c24\u0c4d|\u0dbd\u0d82\u0d9a\u0dcf|\u0e04\u0e2d\u0e21"
+        + "|\u0e44\u0e17\u0e22"
```
Comment (dmdashenkov) [inline client/src/main/java/org/spine3/net/Patterns.java]: Same as above.

File: client/src/main/java/org/spine3/net/Patterns.java [hunk 3]
```diff
@@ -60,103 +60,120 @@
...
-        + "|\u5546\u57ce|\u5546\u5e97|\u5546\u6807|\u5728\u7ebf|\u5927\u62ff|\u5a31\u4e50|\u5de5\u884c"
-        + "|\u5e7f\u4e1c|\u6148\u5584|\u6211\u7231\u4f60|\u624b\u673a|\u653f\u52a1|\u653f\u5e9c"
-        + "|\u65b0\u52a0\u5761|\u65b0\u95fb|\u65f6\u5c1a|\u673a\u6784|\u6de1\u9a6c\u9521|\u6e38\u620f"
-        + "|\u70b9\u770b|\u79fb\u52a8|\u7ec4\u7ec7\u673a\u6784|\u7f51\u5740|\u7f51\u5e97|\u7f51\u7edc"
+        + "|\u4fe1\u606f|\u5065\u5eb7|\u516b\u5366|\u516c\u53f8|\u516c\u76ca|\u53f0\u6e7e"
+        + "|\u53f0\u7063|\u5546\u57ce|\u5546\u5e97|\u5546\u6807|\u5728\u7ebf|\u5927\u62ff"
+        + "|\u5a31\u4e50|\u5de5\u884c|\u5e7f\u4e1c|\u6148\u5584|\u6211\u7231\u4f60|\u624b\u673a"
+        + "|\u653f\u52a1|\u653f\u5e9c|\u65b0\u52a0\u5761|\u65b0\u95fb|\u65f6\u5c1a|\u673a\u6784"
+        + "|\u6de1\u9a6c\u9521|\u6e38\u620f|\u70b9\u770b|\u79fb\u52a8|\u7ec4\u7ec7\u673a\u6784"
+        + "|\u7f51\u5740|\u7f51\u5e97|\u7f51\u7edc"
```
Comment (dmdashenkov) [inline client/src/main/java/org/spine3/net/Patterns.java]: Same as above.

File: server/src/main/java/org/spine3/server/BoundedContext.java [hunk 4]
```diff
@@ -222,7 +222,8 @@ public TenantIndex getTenantIndex() {
      * @param <E>        the type of entities or aggregates
      * @see Repository#initStorage(StorageFactory)
      */
-    @SuppressWarnings("ChainOfInstanceofChecks") // OK here since ways of registering are way too different
+    @SuppressWarnings("ChainOfInstanceofChecks")
+    // OK here since ways of registering are way too different
```
Comment (dmdashenkov) [inline server/src/main/java/org/spine3/server/BoundedContext.java]: Please add the 4 space margin to the comment.

File: client/src/main/java/org/spine3/net/Patterns.java (hunk 1 above)
Comment (dmdashenkov) [inline client/src/main/java/org/spine3/net/Patterns.java]: Let's leave this as is as discussed vocally.

File: server/src/main/java/org/spine3/server/BoundedContext.java (hunk 4 above)
Comment (alexander-aleksandrov) [inline server/src/main/java/org/spine3/server/BoundedContext.java]: Done

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

**Pull request merged**